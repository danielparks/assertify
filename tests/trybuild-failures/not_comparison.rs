use assertify::assertify;

fn main() {
    assertify!(1 + 2);
    unreachable!();
}
