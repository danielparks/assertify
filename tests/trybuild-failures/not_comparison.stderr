error: expected a comparison, e.g. foo() == 1
 --> $DIR/not_comparison.rs:4:16
  |
4 |     assertify!(1 + 2);
  |                ^^^^^
